Assets/Scripts/Game/Tasks/TimerTask.cs

```diff
@@ -16,11 +17,29 @@ public abstract class TimerTask : GameTask
             remainingTime = initialTimerTime;
         }
 
+        /// <summary>
+        /// Determines to a certain condition whether the Timer
+        /// Task should start or not
+        ///
+        /// It can deliver true by default, or be implemented by
+        /// a subclass.
+        /// </summary>
+        /// <returns>true, if that's the case, false otherwise</returns>
+        protected bool TimerStartCondition()
```

Comment:
The method must be declared as `virtual` in order to let sub classes override this behaviour.
```suggestion
        protected virtual bool TimerStartCondition()
```